packages/web/src/room.ts

```diff
@@ -1,78 +1,64 @@
 import {
+  BoxGeometry,
   CircleGeometry,
   FrontSide,
   Group,
   LoadingManager,
-  MathUtils,
   Mesh,
-  MeshPhongMaterial,
+  MeshStandardMaterial,
   NearestFilter,
-  PMREMGenerator,
   RepeatWrapping,
-  Scene,
   Texture,
   TextureLoader,
-  Vector3,
-  WebGLRenderTarget,
-  WebGLRenderer,
 } from "three";
-import { Sky } from "three/examples/jsm/objects/Sky.js";
 
 export class Room {
-  private onLoadCallback: (room: Group) => void;
-  private floorSize = 200;
+  private onLoadCallback: (group: Group) => void;
+
+  private floorSize = 100;
   private floorTexture: Texture | null = null;
   private floorGeometry = new CircleGeometry(this.floorSize, 32);
-  private floorMaterial: MeshPhongMaterial | null = null;
+  private floorMaterial: MeshStandardMaterial | null = null;
   private floorMesh: Mesh | null = null;
 
-  private sky: Sky | null = null;
-  private skyParameters = {
-    elevation: 45,
-    azimuth: 180,
-  };
-  private sunPosition = new Vector3();
-  private pmremGenerator: PMREMGenerator | null = null;
-  private skyRenderTarget: WebGLRenderTarget | null = null;
+  private rampGeometry: BoxGeometry;
+  private rampMaterial: MeshStandardMaterial;
+  private rampMesh: Mesh;
 
   public group: Group = new Group();
 
-  constructor(scene: Scene, renderer: WebGLRenderer, onLoadCallback: (room: Group) => void) {
+  constructor(onLoadCallback: (group: Group) => void) {
     this.onLoadCallback = onLoadCallback;
+
+    this.rampGeometry = new BoxGeometry(4, 15, 0.5);
```

Comment:
I don't think we should keep the ramp in the released version

Reply:
Removed in [commit 5](https://github.com/mml-io/mml-playground/pull/21/commits/01fc1ea646d4c1e9d0eb55bcff3e2de0202b47ff#diff-3476596c0dc13cb1c79245767d26db60d74dd0b54408867424f55833d56ef884L24)